libpod/container_internal_linux.go

```diff
@@ -407,6 +407,14 @@ func (c *Container) generateSpec(ctx context.Context) (*spec.Spec, error) {
 	//nolint:staticcheck
 	g := generate.NewFromSpec(c.config.Spec)
 
+	// If the flag to mount all devices is set for a privileged container, add
+	// all the devices from the host's machine into the container
+	if c.Config().MountAllDevices {
```

Comment:
You can use `c.config` instead of `c.Config()` - it's a little more efficient as `c.Config()` copies the config before returning it.